src/urls.ts

```diff
@@ -50,6 +50,7 @@ const urls = {
 
 export const externalURLs = {
   new_issue: "https://github.com/canonical/juju-dashboard/issues/new",
+  cli_help: "https://juju.is/docs/olm/the-juju-web-cli",
```

Comment:
I have no idea why I originally used snake case here, but would you mind updating both of these links to camel case?

Reply:
Done ✅